app/pb_kits/playbook/pb_nav/_vertical_nav.scss

```
@@ -72,4 +70,4 @@ $selector: ".pb_nav_list";
       }
     }
   }
-}
+}
```

Missing new line here

```suggestion
}

```